core/admin/mailu/ui/templates/token/list.html

```diff
@@ -16,6 +16,7 @@
 {%- call macros.table(order='[[1,"asc"]]') %}
 <thead>
   <tr>
+    <th>{% trans %}ID{% endtrans %}</th>
```

Comment:
I think you should add the ID in the second column and change `1,"asc"` to `2,"asc"` in line 16.
I have started a PR for this "ordering problem", but unfortunately didn't finish it yet.